Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -171,6 +168,18 @@ class TelemetryWrapper {
             object: nil
         )
     }
+    
+    func sendFxADeletionPing() {
+        guard let prefs = profile?.prefs else { return }
+        // Grab our token so we can use the hashed_fxa_uid and clientGUID from our scratchpad for deletion-request ping
+        RustFirefoxAccounts.shared.syncAuthState.token(Date.now(), canBeExpired: false) >>== { (token, kSync) in
```

Comment:
I wonder if `canBeExpired: true` would be useful here. AFAIK the values we're trying to access won't change when you fetch a new token, so getting an expired token should be no big deal and might decrease the chance of something failing while trying to refresh the token here.